src/platform/export/fileConverter.node.ts

```diff
@@ -39,24 +39,7 @@ export class FileConverter extends FileConverterBase implements IFileConverter {
         );
     }
 
-    override async performExport(
```

Comment:
Was this not used?

Reply:
We were overriding it just to change one argument in a method call of the base class.
Rather than copying and pasting the code from the base class, I've added a new method that can be overridden to change the behaviour of what happens when opening the file.